input.go

```diff
@@ -657,12 +686,67 @@ type CheckToolUsageUpdateInput struct {
 	ToolUrlPredicate     *PredicateUpdateInput   `json:"toolUrlPredicate,omitempty" yaml:"toolUrlPredicate,omitempty"`                               // The condition that the tool url should satisfy to be evaluated (Optional)
 }
 
+// CodeIssueIdentifierInput Input for identifying a code issue
+type CodeIssueIdentifierInput struct {
+	CodeIssueProject CodeIssueProjectIdentifierInput `json:"codeIssueProject" yaml:"codeIssueProject"`                               // Identifier of the code issue project to associate with this issue (Required)
+	ExternalId       string                          `json:"externalId" yaml:"externalId" example:"Z2lkOi8vc2VydmljZS8xMjM0NTY3ODk"` // External identifier for this issue (Required)
+}
+
+// CodeIssueInput Input for creating a new code issue
+type CodeIssueInput struct {
+	Cves          []CommonVulnerabilityEnumerationInput `json:"cves,omitempty" yaml:"cves,omitempty" example:"LIST_TODO"`                                // List of CVE identifiers related to this issue (Optional)
+	Cwes          []CommonWeaknessEnumerationInput      `json:"cwes,omitempty" yaml:"cwes,omitempty" example:"LIST_TODO"`                                // List of CWE identifiers related to this issue (Optional)
+	Identifier    CodeIssueIdentifierInput              `json:"identifier" yaml:"identifier"`                                                            // Identifier of the code issue project to associate with this issue (Required)
+	IntroducedAt  *Nullable[iso8601.Time]               `json:"introducedAt,omitempty" yaml:"introducedAt,omitempty" example:"2025-01-05T01:00:00.000Z"` // Timestamp when this issue was introduced (Optional)
+	IssueCategory *Nullable[string]                     `json:"issueCategory,omitempty" yaml:"issueCategory,omitempty" example:"example_value"`          // Category of this code issue. Required to create code issue (Optional)
+	Name          *Nullable[string]                     `json:"name,omitempty" yaml:"name,omitempty" example:"example_value"`                            // Name of the code issue. Required to create code issue (Optional)
+	Severity      *Nullable[string]                     `json:"severity,omitempty" yaml:"severity,omitempty" example:"example_value"`                    // Severity level of this issue (Optional)
+	Url           *Nullable[string]                     `json:"url,omitempty" yaml:"url,omitempty" example:"example_value"`                              // URL with more information about this issue (Optional)
```

Comment:
These need to be vetted - pointers might suffice - not sure if `null` is needed for any of these.